docs/guide/topics/audit_logs.rst

```diff
@@ -0,0 +1,131 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_audit_logs:
+
+Audit Logs
+============
+
+Audit logs log various administrative actions taken in a guild. Examples include:
```

Comment:
If we're just saying "include", I don't think we should include **everything** that gets logged. We can either reduce the list down to be readable or change this to specify that it's all of them.